azavea/pwd-fishway
# Add quiz view

## Overview

When the user elects to play the quiz, the user is dropped into a fullscreen quiz outside the framework of the 4 tabs. When the quiz is exited, whether by finishing or closing out, we still want to return to the user to the quiz tab and previous app state. This PR adds the quiz view and implements that behavior.

Connects #40 

### Demo

![quizinternal](https://user-images.githubusercontent.com/10568752/57399718-d2176c80-719f-11e9-9c99-5a0a13f8bb8e.gif)


### Notes

Our root folder is growing with components, so I organized them into a `components` subdirectory. We may want a separate folder for styled-components.


## Testing Instructions

Use the Netlify site and navigate to the quiz page. Click Play and then the "close" exit. See the you're sent back to QuizHome and the About state is maintained
